application/helpers/expressions/em_manager_helper.php

```diff
@@ -9800,6 +9752,81 @@ public static function htmlSpecialCharsUserValue($string)
         $string = str_replace(["{","}"], ["&#123;","&#125;"], $string);
         return $string;
     }
+
+    /**
+     * Save the starting values to the database
+     */
+    public static function SaveStartingValues()
+    {
+        // Init
+        $LEM =& LimeExpressionManager::singleton();
+
+        // If not starting values, retun
+        if (!isset($_SESSION[$LEM->sessid]['startingValues'])) {
+            return false;
+        }
+        if (!is_array($_SESSION[$LEM->sessid]['startingValues'])) {
+            return false;
+        }
+        if (!array_key_exists('startingValues', $_SESSION[$LEM->sessid])) {
```

Comment:
Question : it's not the same than `if (!isset($_SESSION[$LEM->sessid]['startingValues'])) {` ?

We can not remove this tes ?

Reply:
They are very similar. Isset checks for NULL values.
https://www.geeksforgeeks.org/difference-between-isset-and-array_key_exists-function-in-php

I will remove it.